doc/template.html

```diff
@@ -45,5 +45,6 @@
   <script src="assets/sh_main.js"></script>
   <script src="assets/sh_javascript.min.js"></script>
   <script>highlight(undefined, undefined, 'pre');</script>
+  __TRACKING__
```

Comment:
@jbergstroem it gets replaced w/empty string when tracking is disabled, which is default, so it shouldn't be needed to enclose in comments?